packages/portfolio/src/adapters/dullahan-api/api-maker.ts

```diff
@@ -14,11 +14,13 @@ import {
   toProcessedMediaRequest,
   toSecondaryTokenInfos,
   toTokenActivityUpdates,
+  toTokenHistoryUpdates,
```

Comment:
only price on this endpoint, I think it can be tokenPriceHistory
```suggestion
  toTokenPriceHistory,
```

Reply:
I went with TokenHistory instead of TokenPriceHistory everywhere, should be easy to change but I don't think we'll have any other kind of token history